hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestDataGenerator.java

```diff
@@ -681,6 +681,10 @@ public List<HoodieRecord> generateInserts(String instantTime, Integer n) {
     return generateInserts(instantTime, n, false);
   }
 
+  public List<HoodieRecord> generateInserts(Long instantTime, Integer n) {
+    return generateInserts(String.valueOf(instantTime), n, false);
+  }
```

Comment:
Could caller just use String type instead of adding redundant test methods here?

Reply:
I've removed all new test methods in this class